build-support/bin/ci.py

```diff
@@ -417,19 +478,60 @@ def run_jvm_tests() -> None:
   )
 
 
-def run_integration_tests(*, shard: Optional[str]) -> None:
+def run_integration_tests_v1(*, shard: Optional[str]) -> None:
   check_pants_pex_exists()
-  all_targets = get_all_python_tests(tag="+integration")
+  target_sets = TestTargetSets.calculate(
+    test_type=TestType.integration,
+    default_behavior=DefaultTestBehavior.v1_no_chroot,
+    remote_execution_enabled=False
+  )
   command = ["./pants.pex", "test.pytest"]
   if shard is not None:
     command.append(f"--test-pytest-test-shard={shard}")
   with travis_section("IntegrationTests", f"Running Pants Integration tests {shard if shard is not None else ''}"):
     try:
-      subprocess.run(command + sorted(all_targets) + PYTEST_PASSTHRU_ARGS, check=True)
+      subprocess.run(
+        command + ["--test-pytest-chroot"] + sorted(target_sets.v1_chroot) + PYTEST_PASSTHRU_ARGS,
+        check=True
+      )
+      subprocess.run(command + sorted(target_sets.v1_no_chroot) + PYTEST_PASSTHRU_ARGS, check=True)
```

Comment:
Maybe have separate travis sections here, the same way we do for v2? 

I'm less worried about this because these sets should be very fluid and the user shouldn't care that much, but I think that it would improve readability.

Reply:
Done.